medical_physician/models/medical_physician.py

```diff
@@ -38,6 +38,17 @@ class MedicalPhysician(models.Model):
         comodel_name='medical.physician.schedule.template',
         inverse_name='physician_id',
     )
+    medical_center_primary_id = fields.Many2one(
```

Comment:
The existing medical center concept was not supposed to be changed. I believe it used to be parent_id